libs/common/src/types/mongo-db-models/facility.ts

```diff
@@ -42,4 +42,7 @@ export type Facility = {
   hasBeenIssuedAndAcknowledged: boolean | null;
   canResubmitIssuedFacilities: boolean | null;
   unissuedToIssuedByMaker: object;
+  isUsingFacilityEndDate: boolean | null;
+  bankReviewDate: IsoDateTimeStamp | Date | null;
+  facilityEndDate: IsoDateTimeStamp | Date | null;
```

Comment:
What's the reason for doing `IsoDateTimeStamp | Date` as opposed to just `Date`?

Reply:
This was me not being confident in mongodb, been down a rabbit hole & happy to say it is a Date